doc/users/prev_whats_new/whats_new_3.6.0.rst

```diff
@@ -69,6 +70,7 @@ Using the ``layout='compressed'`` layout reduces the space between the Axes,
 and adds the extra space to the outer margins:
 
 .. plot::
+    :alt: Four identical two by two heatmaps, each cell a different color: purple, blue, yellow, green going clockwise from upper left corner. The four heatmaps are laid out in a two by two grid with minimum white space between the heatmaps.
```

Comment:
I still think these should wrap, unless that is not possible with the alt tag?

Reply:
@jklymak They appear wrapped- was that the clarification you were asking for?

Reply:
Unfortunately, it does not appear possible to wrap these lines. I even tried with a backslash escape, but Sphinx didn't like it either way.